task-sdk/tests/task_sdk/execution_time/test_task_runner.py

```diff
@@ -864,18 +832,7 @@ def test_run_with_asset_inlets(create_runtime_ti, mock_supervisor_comms):
                     AssetProfile(name="name", uri="s3://bucket/my-task", type="Asset"),
                     AssetProfile(name="new-name", uri="s3://bucket/my-task", type="Asset"),
                 ],
-                outlet_events=[
-                    {
-                        "asset_alias_events": [],
-                        "extra": {},
-                        "key": {"name": "name", "uri": "s3://bucket/my-task"},
-                    },
-                    {
-                        "asset_alias_events": [],
-                        "extra": {},
-                        "key": {"name": "new-name", "uri": "s3://bucket/my-task"},
-                    },
-                ],
+                outlet_events=[],
```

Comment:
Why do we remove the events?

Reply:
`outlet_events` now only include entries that don’t have the default extra. None of these have custom extras so they are not included in this list anymore. There is a test making sure outlets without an entry in `outlet_events` are still emitted correctly.

Reply:
got it. thanks!